lib/bemxjst/index.js

```diff
@@ -426,13 +426,13 @@ BEMXJST.prototype.tryRun = function(context, ent) {
   try {
     return ent.run(context);
   } catch (e) {
-    console.error('BEMXJST ERROR: cannot render ' +
-      [
-        'block ' + context.block,
-        'elem ' + context.elem,
-        'mods ' + JSON.stringify(context.mods),
-        'elemMods ' + JSON.stringify(context.elemMods)
-      ].join(', '), e);
+    var error = this.context.onError || console.error;
+    error({
+      block: context.block,
+      elem: context.elem,
+      mods: context.mods,
+      elemMods: context.elemMods
```

Comment:
1. Looks like it's not the same as it was. Why we decided to change it?
2. Why we can't pass context object as the first argument?